### DISCUSSION TITLE
underground geometry got clips

### DISCUSSION THREAD

**Discussion by tan9710630** [visgl/deck.gl#6530] [Q&A] [answered]
I build a custom layer to show underground tubes, I set `position.z` under 0.
It work just fine, but when I move camera close to the tube geometry, it seems like camera clips the geometry, something like this:
![111111111111 00_00_00-00_00_30](https://user-images.githubusercontent.com/22703484/147345939-7cb3df30-81ae-4bab-91a9-4a8eecbdd5b3.gif)
Anyone know how to fix this error?

**Comment by Pessimistress** [comment, marked as answer]
[farZMultiplier](https://deck.gl/docs/api-reference/core/map-view#farzmultiplier)

**Reply by tan9710630** [reply]
thanks a lot, It works!!!!!
